integrations/content.yml

```diff
@@ -27,3 +29,9 @@ features:
   - Vue components in Markdown
   - Full-text search
   - Table of contents generation
+articles:
+  - name: Create a Blog with Nuxt Content
```

Comment:
What do you think about title instead of name?

Reply:
ok